appveyor.yml

```diff
@@ -0,0 +1,46 @@
+version: '{build}'
+os: Windows Server 2012
+install:
+  - ps: iex (new-object net.webclient).downloadstring('https://get.scoop.sh')
+  - ps: scoop install sbt
+  - ps: $env:SBT_OPTS="-XX:MaxPermSize=2g -Xmx4g"
+  - ps: $env:CLANG_PATH="C:\\Program Files\\LLVM\\bin\\clang.exe"
+  - ps: $env:CLANGPP_PATH="C:\\Program Files\\LLVM\\bin\\clang++.exe"
+  - ps: $env:CLANG_FORMAT_PATH="C:\\Program Files\\LLVM\\bin\\clang-format.exe"
+  - ps: $env:SCALA_NATIVE_USER_INCLUDES_PATH="$pwd\os_win"
+  - ps: $env:JAVA_TOOL_OPTIONS="-Dfile.encoding=UTF8"
+  - ps: $env:JAVA_OPTS="-Xmx2G"
+build_script:
+  - ps: |
+        Write-Host $env:APPVEYOR_REPO_NAME;
+        if ($env:APPVEYOR_PULL_REQUEST_NUMBER) {
+            Write-Output "Incoming pull request from https://github.com/$env:APPVEYOR_REPO_NAME/pull/$env:APPVEYOR_PULL_REQUEST_NUMBER";
+            $authorReq= try { Invoke-RestMethod -UseDefaultCredentials -Uri "https://api.github.com/repos/$env:APPVEYOR_REPO_NAME/pulls/$env:APPVEYOR_PULL_REQUEST_NUMBER" -ErrorVariable RestError -ErrorAction SilentlyContinue} catch { $RestError.message | ConvertFrom-Json};
+            $author = $authorReq.user.login;
+            if ($author) {
+                Write-Output "Pull request submitted by $author";
+                $signedReq=try { Invoke-RestMethod -Uri "http://www.lightbend.com/contribute/cla/scala/check/$author" -ErrorVariable RestError -ErrorAction SilentlyContinue} catch { $RestError.message | ConvertFrom-Json};
+                if ($signedReq.signed -eq "true") {
+                    Write-Output "CLA check for $author successful";
+                }
+                else {
+                    Write-Output "CLA check for $author failed";
+                    Write-Output "Please sign the Scala CLA to contribute to Scala Native";
+                    Write-Output "Go to https://www.lightbend.com/contribute/cla/scala and then resubmit this pull request";
+                    exit 1;
+                } else {exit 1};
+            } else {exit 1};
+        }
```

Comment:
We already test for CLA in the Travis CI, there is no need to duplicate the logic again in the Windows CI. If the CLA wasn't signed, travis build should fail.